### ISSUE TITLE
Loading a jsTestDriver.conf file that references files in the current working directory throws NullPointerException

### ISSUE THREAD

**Issue by GoogleCodeExporter** [manojjsm/js-test-driver#196]
```
What steps will reproduce the problem?
1. Create a config file like below:

load:
  - my_test.js

2. Attempt to run the tests

What is the expected output?
The test should run and display appropriate output.

What do you see instead?
java.lang.NullPointerException
    at com.google.jstestdriver.FileInfo.getPath(FileInfo.java:185)
    at com.google.jstestdriver.PathResolver.resolve(PathResolver.java:133)
    at com.google.jstestdriver.config.ParsedConfiguration.resolvePaths(ParsedConfiguration.java:78)
    at com.google.jstestdriver.config.Initializer.initialize(Initializer.java:81)
    at com.google.jstestdriver.JsTestDriver.main(JsTestDriver.java:79)
Unexpected Runner Condition: null
 Use --runnerMode DEBUG for more information.

Please use labels and text to provide additional information.

```

Original issue reported on code.google.com by `robertsd...@gmail.com` on 9 Feb 2011 at 3:11

**Comment by GoogleCodeExporter** [comment]
Original comment by `corbinrs...@gmail.com` on 18 Feb 2011 at 5:40
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
In what version is this fixed? I see the problem in JsTestDriver-1.3.1.jar.
```

Original comment by `ccalo@google.com` on 15 Mar 2011 at 3:17
